### ISSUE TITLE
Space Mode: Export to MiSTer copied everything then deleted it

### ISSUE THREAD

**Issue by matijaerceg** [city41/AMMiSTer#94]
Seems that the other issue where Space Mode was sometimes deleting on export isn't resolved. This is with 0.25.4 on Win10

**Comment by matijaerceg** [comment]
One last note for the day: just did an export using Speed mode, and a bunch of stuff is also missing and/or deleted. Hopefully easy to repro for you on Windows.

**Comment by matijaerceg** [comment]
Just ran it twice, and both times it seemed to not delete stuff, but the weird thing this time is that even though it's space mode, it didn't delete the stuff that was already there that should have gotten deleted.

logs attached

[export-log--exportToMister-New_Plan-2023-04-17T16_40_03.652Z.zip](https://github.com/city41/AMMiSTer/files/11253128/export-log--exportToMister-New_Plan-2023-04-17T16_40_03.652Z.zip)

**Comment by city41** [comment]
Thanks. Just quickly looking at the logs I see several issues, they mostly boil down to windows using `\` and mister using `/` for path separator.

**Comment by city41** [comment]
I'm just going to switch Windows to `/`. Initially `\` was simple and I chose it to make the app feel a little bit more familiar with Windows users. But it's just not worth it. Windows and Node on Windows both support `/`, and this will remove a lot of Windows specific code from the app.

I can always have it switch from `/` to `\` in the details modal if needed.

**Comment by matijaerceg** [comment]
That sounds wise. I didn't even notice slashes in the program interface tbh

**Comment by city41** [comment]
Hoo this is painful. I'll always curse Microsoft for doing this :) 

I think I have this working. A huge downside is all existing plans created on Windows will no longer work. I think it's best just to rip that band aid off though.

The upside is the code is still a little crazy, but not nearly as bad as it was before.

**Comment by matijaerceg** [comment]
Yeah, it's early days, I don't think you should feel bad about breaking existing plans at all.

**Comment by city41** [comment]
I have released [0.25.7](https://github.com/city41/AMMiSTer/releases/tag/v0.25.7). In testing I could not find anything wrong, but there may still be some minor issues.

catalog.json needs to be recreated for Windows users, noted in the release notes.

With this, AMMiSTer on Windows is now pretty much the same as on Linux and Mac. So one interesting upside: plan files can now work across OSes.

**Comment by matijaerceg** [comment]
I think this is great, and will hopefully make multi-platform development easier going forward. Thanks for the updates, I'll try this on my windows machine hopefully tonight and let you know how it goes.

**Comment by matijaerceg** [comment]
Confirming this is fixed for me Win 10 v0.26.0
